health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -567,30 +589,44 @@ const UpdateChecklist = () => {
+                                }}
+                                onClose={() => setShowLocalisationPopup(false)}
+                            />
+                        </PopUp>
+                    )}
+
 
 
                 </div>
             }
-            <div style={{height: "2rem"}}></div>
+            <div style={{ height: "2rem" }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)